config/initializers/sentry.rb

```diff
@@ -8,4 +8,22 @@
   # of transactions for performance monitoring.
   # We recommend adjusting this value in production
   config.traces_sample_rate = 0.25
+
+  exception_fingerprints = {
+    Excon::Error::ServiceUnavailable => 'external-service',
+    Twitter::Error::InternalServerError => 'external-service',
+  }
+  config.before_send do |event, hint|
+    if hint[:exception].is_a?(Errors::Base)
+      # These are used for user-facing errors, not when something goes wrong
+      nil
```

Comment:
use `next` to return from a block early
```suggestion
      next nil
```